src-docs/src/views/context_menu/context_menu_example.js

```diff
@@ -122,6 +123,22 @@ export const ContextMenuExample = {
             <EuiCode language="ts">width: [number of pixels]</EuiCode> to the
             panel tree.
           </p>
+          <p>
+            You can add separator lines through <EuiCode>items</EuiCode> prop if
+            you define items as{' '}
+            <EuiCode language="ts">{'{isSeparator: true}'}</EuiCode>. Separator
+            passes through rest of its fields as props to{' '}
+            <Link to="/layout/horizontal-rule">
+              <strong>EuiHorizontalRule</strong>
+            </Link>{' '}
+            component. Separators can also have a{' '}
+            <EuiLink
+              href="https://reactjs.org/docs/lists-and-keys.html#keys"
+              target="_blank">
+              <EuiCode>key</EuiCode> prop
+            </EuiLink>
+            , used by React for list rendering.
+          </p>
```

Comment:
This last line shouldn't be necessary because it's pretty much true for all components/elements and you already mention that the rest of props will pass through (which would include `key`).
```suggestion
            component.
          </p>
```

Reply:
Removed.